### PR TITLE
Reformat UG

### PR DESCRIPTION
**Pull request by eugenetayyj**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -141,26 +153,27 @@ Format: `create n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [ds/DESCRIPTION] [nw
...
 :bulb: **Note:**
-It is recommended to only have 1`TAG` per client.
+- `NAME` and `PHONE_NUMBER` are the only compulsory inputs. Other parameters can be left blank.
+- It is recommended to only have 1`TAG` per client.
                   
-
-* If you have multiple meeting times with your client, simply repeat the field `mt/TIME`.
-* Does not allow you to create a person with the same name and phone number as a current person in the FABook.
-* Allows you to create a person with same name but different phone number or same number and different name as a current person in the FABook.
+</div>
```
Comment (ReaganTan00) [inline docs/UserGuide.md:162]: I think we should use a different colour to separate things like Notes and Command Shortcuts. 
I made the changes according to this in my PR but open to discussion

File: docs/UserGuide.md [hunk 2]
```diff
@@ -240,12 +268,18 @@ Format: `find a/ADDRESS`
...
 Format: `f a/ADDRESS`
 
+</div>
+
 #### Finding your client by tag : `find`
 
-Find your clients whose tag matches the input tag.
+Imagine you want to consolidate the clients you have `SECURED` and share with them the new perks your company has rolled out. Or you want to find all `POTENTIAL` clients so you can make a concerted and targeted effort to convert them into customers. Then this is the command you will want to run.
+
+You can find your clients whose tag matches the input tag.
```
Comment (ReaganTan00) [inline docs/UserGuide.md:282]: Good purpose/use case. 
Needs to be added to all commands?

**Review by ReaganTan00 — APPROVED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -52,6 +52,13 @@ This user guide will help you get started and understand how FABook can **seamle
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Legend
+1. :white_check_mark: : **Input Shortcut**
+2. :bulb: :  **Note**
+3. :heavy_exclamation_mark: : **Caution**
```
Comment (fungusta) [inline docs/UserGuide.md:58]: Lovely legend

**Review by fungusta — APPROVED** [review]
LGTM

**Pull request merged**